backend/api/scraper_handler.go

```diff
@@ -0,0 +1,280 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/AustrianDataLAB/GeWoScout/backend/cosmos"
+	"github.com/AustrianDataLAB/GeWoScout/backend/models"
+	"github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos"
+	"github.com/go-chi/render"
+	"github.com/xeipuuv/gojsonschema"
+	"golang.org/x/text/runes"
+	"golang.org/x/text/transform"
+	"golang.org/x/text/unicode/norm"
+	"log"
+	"net/http"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+)
+
+type bindingInput struct {
+	Data     bindingData     `json:"Data"`
+	Metadata bindingMetadata `json:"Metadata"`
+}
+
+type bindingData struct {
+	Msg string `json:"msg"`
+}
+
+type bindingMetadata struct {
+	Id           string `json:"Id"`
+	DequeueCount string `json:"DequeueCount"`
+}
```

Comment:
**suggestion**: Could we move these to the models package? I think it makes sense, since we have the other input binding structs there as well